src/otx/algo/classification/efficientnet.py

```diff
@@ -106,9 +104,8 @@ def _build_model(self, num_classes: int) -> nn.Module:
             head=LinearClsHead(
                 num_classes=num_classes,
                 in_channels=backbone.num_features,
-                topk=(1, 5) if num_classes >= 5 else (1,),
-                loss=loss,
             ),
+            loss=nn.CrossEntropyLoss(),
```

Comment:
Yes, you are correct. Instead of dividing by `cls_score.size(0)` in the `loss function` like we did before, i changed it to use mean by default, so the behavior is the same.